crates/runtime/tests/spiceai/mod.rs

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2024-2025 The Spice.ai OSS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+use std::sync::Arc;
+
+use app::AppBuilder;
+use futures::StreamExt;
+use runtime::{status, Runtime};
+use spicepod::component::{dataset::Dataset, params::Params};
+
+use crate::{
+    get_test_datafusion, init_tracing, run_query_and_check_results, utils::test_request_context,
+    ValidateFn,
+};
+
+fn make_spiceai_dataset(path: &str, name: &str) -> Dataset {
+    let mut dataset = Dataset::new(format!("spice.ai/{path}"), name.to_string());
+    dataset.params = Some(Params::from_string_map(
+        vec![
+            (
+                "spiceai_api_key".to_string(),
+                "31383639|bf7394eae35d4e85942126583facdf82".to_string(),
+                // "${ env:SPICEAI_API_KEY }".to_string(),
```

Comment:
```suggestion
                "${ env:SPICEAI_API_KEY }".to_string(),
```

Reply:
That is my personal test spice app, so nothing critical exposed